src/Atmos/Model/ref_state.jl

```diff
@@ -210,31 +153,41 @@ Tv(z) = \\max(Tv{\\text{surface}} − ΔTv \\tanh(z/H_{\\text{t}})
 
 $(DocStringExtensions.FIELDS)
 """
-struct DecayingTemperatureProfile{FT} <: TemperatureProfile
-    "virtual temperature at surface (K)"
-    Tv_surface::FT
-    "virtual temperature drop from surface to top of the atmosphere (K)"
+struct DecayingTemperatureProfile{FT} <: TemperatureProfile{FT}
+    "Virtual temperature at surface (K)"
+    T_virt_surf::FT
+    "Virtual temperature drop from surface to top of the atmosphere (K)"
     ΔTv::FT
-    "height scale over which virtual temperature drops (m)"
+    "Height scale over which virtual temperature drops (m)"
     H_t::FT
 end
+function DecayingTemperatureProfile(
+    param_set::AbstractParameterSet,
+    T_virt_surf::FT,
+    ΔTv::FT,
+) where {FT <: AbstractFloat}
+    _R_d::FT = R_d(param_set)
+    _grav::FT = grav(param_set)
+    H_t = _R_d * T_virt_surf / _grav
```

Comment:
> 1. Have H_t as a genuine input parameter of DecayingTemperatureProfile

I thought that you asked for `H_t` to be derived from the other arguments?

> 2. Make surface T_surf_ref and ΔTv_ref CIMAParameters for Earth.

Opened [18](https://github.com/CliMA/CLIMAParameters.jl/pull/18) in CLIMAParameters

> Then set DecayingTemperatureProfile with `T_virt_surf = T_surf_ref, ΔTv = ΔTv_ref, and H_t = _R_d * T_surf_ref / grav` on the driver level.

This seems to conflict with a different comment you left:

> If the default behavior for DecayingTemperatureProfile is to use `H_t = _R_d * T_sfc / _grav`, when no `H_t` is given, we can simply omit it here. No need for redundant specifications.

Maybe we can review this interactively?

Reply:
> > 1. Have H_t as a genuine input parameter of DecayingTemperatureProfile
> 
> I thought that you asked for `H_t` to be derived from the other arguments?

The easiest option, and I thought the one we agreed upon, is to have `H_t = R_d * T_surf/g` when nothing else is given (default), but have the option to override that when needed.

> > 1. Make surface T_surf_ref and ΔTv_ref CIMAParameters for Earth.
> 
> Opened [18](https://github.com/CliMA/CLIMAParameters.jl/pull/18) in CLIMAParameters
> 
> > Then set DecayingTemperatureProfile with `T_virt_surf = T_surf_ref, ΔTv = ΔTv_ref, and H_t = _R_d * T_surf_ref / grav` on the driver level.

Yes, my original suggestion was to set all three as parameters. Then you implemented the scale height in the `DecayingTemperatureProfile`, which is a fine alternative if it is an *option* (not a necessity) there. So then I said just specify `T_virt_surf` and `ΔTv` as CLIMAParameters. 
> > If the default behavior for DecayingTemperatureProfile is to use `H_t = _R_d * T_sfc / _grav`, when no `H_t` is given, we can simply omit it here. No need for redundant specifications.

The statement here is that if `DecayingTemperatureProfile` uses  `H_t = _R_d * T_sfc / _grav` as the default when no `H_t` is given, there is no point to specify in multiple places
`H_t = _R_d * T_sfc / _grav` as a (redundant) input argument.

So in short, here's what works: write the code with `H_t` as optional input argument. Without it, it defaults to the scale height RT/g. But this can be overridden by other height scales specified by a user. 

Interactive would be good. But I'm really busy tonight and all of tomorrow. If the concepts are unclear, perhaps @szy21 can help explain.

Reply:
I'm happy to talk tonight if needed.

Reply:
Ok, yes, sorry for the confusion. Now, there is a single `DecayingTemperatureProfile` method with optional arguments

 - T_virt_surf
 - ΔTv
 - H_t

Once ΔTv and T_virt_surf are merged into CLIMAParameters, I'll make these values default to those defined in CLIMAParameters.